src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java

```diff
@@ -52,6 +53,31 @@ public void onBlockBreak(final BlockBreakEvent event) {
 
         this.grantAdvancement(player, key);
     }
+    
+        @EventHandler
+    public void onPlayerStatisticIncrement(final PlayerStatisticIncrementEvent event) {
+        // Check if it's a diamond ore
+        if (event.getStatistic() != statistic.FLY_ONE_CM) {
```

Comment:
You need to use the enum here: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Statistic.html